src/order/basic.lean

```diff
@@ -191,6 +191,51 @@ instance (α : Type*) [decidable_linear_order α] : decidable_linear_order (orde
 
 instance : Π [inhabited α], inhabited (order_dual α) := id
 
+/-- `to_dual` is the identity function to the `order_dual` of a linear order.  -/
+def to_dual : α → order_dual α := id
+
+/-- `of_dual` is the identity function from the `order_dual` of a linear order.  -/
+def of_dual : order_dual α → α := id
```

Comment:
Can you make this an equiv?
```suggestion
/-- `to_dual` is the identity function to the `order_dual` of a linear order.  -/
def to_dual : α ≃ order_dual α := ⟨id, id, rfl, rfl⟩

/-- `of_dual` is the identity function from the `order_dual` of a linear order.  -/
def of_dual : order_dual α ≃ order_dual α := to_dual.symm
```

Reply:
I am happy to accept this change, although I do not really understand the implications (nor the commands)!

Reply:
The advantage of this change is that if I want to use a lemma that requires an equivalence (eg `finsupp.dom_congr`), then I can use `to_dual` directly rather than having to convert it to an equivalence myself.

Reply:
Lean complains about the symbol \equiv: it says "unexpected token".  How should I type it?

Reply:
(Typing \equiv converts to the right symbol, but Lean does not like it...)

Reply:
You'll need to import data.equiv.basic